### PR TITLE
macOS related fixes

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
See title.

### CODE REVIEW

**Review by garethgreenaway — APPROVED** [review]

File: bootstrap-salt.sh [hunk 1]
```diff
@@ -6750,6 +6750,12 @@ install_macosx_stable_deps() {
 install_macosx_git_deps() {
     install_macosx_stable_deps || return 1
 
+    echo $PATH | grep -q /usr/local/bin
```
Comment (vutny) [inline bootstrap-salt.sh]: Need to quote the `"$PATH"` I guess...

Comment (s0undt3ch) [reply bootstrap-salt.sh]: Fixed

File: bootstrap-salt.sh [hunk 2]
```diff
@@ -6750,6 +6750,12 @@ install_macosx_stable_deps() {
 install_macosx_git_deps() {
     install_macosx_stable_deps || return 1
 
+    echo "$PATH" | grep -q /usr/local/bin
+    if [  $? -eq 1 ]; then
```
Comment (vutny) [inline bootstrap-salt.sh]: How about to include the whole command into the `if` statement expression?
No need for additional return code check:
```suggestion
    if ! echo "$PATH" | grep -q /usr/local/bin; then
```

File: bootstrap-salt.sh [hunk 3]
```diff
@@ -6750,6 +6750,12 @@ install_macosx_stable_deps() {
 install_macosx_git_deps() {
     install_macosx_stable_deps || return 1
 
+    echo "$PATH" | grep -q /usr/local/bin
```
Comment (vutny) [inline bootstrap-salt.sh]: This line no longer needed :smiley: 
```suggestion
```

**Review by vutny — APPROVED** [review]
Good job! I guess this needs rebasing... and CI have passed on previous iteration anyway.

**Comment by s0undt3ch** [comment]
Done! :smile:

**Comment by vutny** [comment]
CI seems OK, just Opensuse being unstable as usual...

**Pull request merged**